### ISSUE TITLE
RGBShiftShader: Broken URL

### ISSUE THREAD

**Issue by Davidyanlong** [mrdoob/three.js#25477]
### Description

https://github.com/mrdoob/three.js/tree/master/examples/jsm/shaders/RGBShiftShader.js

comment www address (http://kriss.cx/tom/ ) is not available
/**
 * RGB Shift Shader
 * Shifts red and blue channels from center in opposite directions
 * Ported from http://kriss.cx/tom/2009/05/rgb-shift/
 * by Tom Butterworth / http://kriss.cx/tom/
 *
 * amount: shift distance (1 is width of input)
 * angle: shift angle in radians
 */

### Reproduction steps

1.  visit to http://kriss.cx/tom/  not visit
2. visit to http://kriss.cx/tom/2009/05/rgb-shift/
3.

### Code

```js
/**
 * RGB Shift Shader
 * Shifts red and blue channels from center in opposite directions
 * Ported from http://kriss.cx/tom/2009/05/rgb-shift/
 * by Tom Butterworth / http://kriss.cx/tom/
 *
 * amount: shift distance (1 is width of input)
 * angle: shift angle in radians
 */
```

### Live example

https://github.com/mrdoob/three.js/tree/master/examples/jsm/shaders/RGBShiftShader.js

### Version

current version

**Comment by gkjohnson** [comment]
The link contents can be found on the wayback machine: https://web.archive.org/web/20090820185047/http://kriss.cx/tom/2009/05/rgb-shift/

**Comment by Mugen87** [comment]
Should be fixed via f22aae7b6b4c42f7e3ecccd5e57a421fda39d2eb.
